src/data/moves/move.ts

```diff
@@ -861,7 +863,9 @@ export abstract class Move {
 
     let attrs: MoveAttr[] = this.attrs;
     if (isKnockOut) {
-      attrs = attrs.filter((attr) => attr.selfTarget);
+      attrs = attrs.filter(
```

Comment:
Should probably add a comment explaining what this filter is doing.
```suggestion
      // <comment goes here>
      attrs = attrs.filter(
```